Assets/Scripts/LevelSelection.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class LevelSelection : MonoBehaviour
+{
+    public Button[] lvlButtons;
+
+    // Start is called before the first frame update
```

Comment:
This comment can be removed. (The comments in `LevelEndTriggerController.cs` can also be removed.)